CHANGELOG.md

```diff
@@ -9,6 +9,7 @@
 
 **Bugfixes**:
 
+* Search: Normalise search button styles and use 44px height. Removed `cads-button` from search input.
```

Comment:
This needs to go at the top of the files, before the version for the previous release.